package.json

```
@@ -1,5 +1,22 @@
 {
+  "scripts": {
+    "dev": "tsc --watch",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix",
+    "format": "prettier --write src/"
```

```suggestion
    "build": "tsc --strict",
    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --ignore-path .gitignore --fix",
    "type-check": "tsc --strict --noEmit",
    "lint-check": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --ignore-path .gitignore"
```
Remove the "format" script as eslint in `npm run lint` fixes issues identified in prettier. Add a type-check script that's required in CI/CD.
Add a lint-check that's required in CI/CD.
Effectively, running `npm run lint` should format the project.
Instead of having a separate `.eslintignore`, we should just be using our `.gitignore`